src/main/scala/org/broadinstitute/dsde/workbench/leonardo/service/LeonardoService.scala

```diff
@@ -92,6 +93,7 @@ case class InvalidDataprocMachineConfigException(errorMsg: String)
   extends LeoException(s"${errorMsg}", StatusCodes.BadRequest)
 
 class LeonardoService(protected val dataprocConfig: DataprocConfig,
+                      protected val welderDao: WelderDAO,
```

Comment:
I think there are lots of compilation errors due to this new field (I'm looking at travis results).

Reply:
ah yea, I haven't fix tests yet